nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -728,14 +767,34 @@ in {
               ${optionalString (c.dbport != null) "'dbport' => '${toString c.dbport}',"}
               ${optionalString (c.dbuser != null) "'dbuser' => '${c.dbuser}',"}
               ${optionalString (c.dbtableprefix != null) "'dbtableprefix' => '${toString c.dbtableprefix}',"}
-              ${optionalString (c.dbpassFile != null) "'dbpassword' => nix_read_secret('${c.dbpassFile}'),"}
+              ${optionalString (c.dbpassFile != null) ''
+                  dbpassword' => nix_read_file(
```

Comment:
Shouldn't this be `nix_read_secret`?

Reply:
it should be, yes! seems to have been forgotten at some point in this PR's history …